src/plugins/gui/fiatPluginTypes.ts

```diff
@@ -49,6 +49,8 @@ export const asFiatPaymentType = asValue(
   'directtobank',
   'fasterpayments',
   'googlepay',
+  // TODO: Remove `iach` from the variants when old fiat plugin code has been gutted.
+  /** @deprecated because Instant ACH isn't different from regular ACH. Use `ach` instead. */
   'iach',
```

Comment:
This value isn't actually deleted yet? Either delete the constant, or put another comment back like `// The info server still sends this, but we don't use it anymore`

Reply:
See the TODO. It mentions why it's not deleted.